crates/gitbutler-core/src/project_repository/repository.rs

```diff
@@ -611,6 +611,12 @@ impl Repository {
     pub fn remotes(&self) -> Result<Vec<String>> {
         self.git_repository.remotes().map_err(anyhow::Error::from)
     }
+
+    pub fn add_remote(&self, name: &str, url: &str) -> Result<()> {
+        self.git_repository
```

Comment:
Another common way of writing this is `Ok(self.git_repository.add_remote(name, url)?)`, which is usually preferred as it turns into a one-liner. Once it breaks the line, the form used here is usually nicer to look at.

Reply:
I did look at `Ok(<expression>?)` when writing but it felt super jarring in terms of control flow perspective. I'll keep that wrapping heuristic though and make a change tomorrow morning :+1: